objects/src/testing.rs

```diff
@@ -0,0 +1,57 @@
+use vm_core::{code_blocks::CodeBlock, Operation, Program, ZERO};
+
+use self::constants::NON_FUNGIBLE_ASSET_DATA_2;
+use crate::{
+    accounts::AccountId,
+    assets::{Asset, NonFungibleAsset, NonFungibleAssetDetails},
+};
+
+pub mod constants {
+    use vm_core::Felt;
+
+    pub const FUNGIBLE_ASSET_AMOUNT: u64 = 100;
+    pub const FUNGIBLE_FAUCET_INITIAL_BALANCE: u64 = 50000;
+
+    pub const MIN_PROOF_SECURITY_LEVEL: u32 = 96;
+
+    pub const CONSUMED_ASSET_1_AMOUNT: u64 = 100;
+    pub const CONSUMED_ASSET_2_AMOUNT: u64 = 200;
+    pub const CONSUMED_ASSET_3_AMOUNT: u64 = 300;
+    pub const CONSUMED_ASSET_4_AMOUNT: u64 = 100;
+
+    pub const NON_FUNGIBLE_ASSET_DATA: [u8; 4] = [1, 2, 3, 4];
+    pub const NON_FUNGIBLE_ASSET_DATA_2: [u8; 4] = [5, 6, 7, 8];
+
+    pub const CHILD_ROOT_PARENT_LEAF_INDEX: u8 = 10;
+    pub const CHILD_SMT_DEPTH: u8 = 64;
+    pub const CHILD_STORAGE_INDEX_0: u64 = 40;
+    pub const CHILD_STORAGE_VALUE_0: [Felt; 4] =
+        [Felt::new(11), Felt::new(12), Felt::new(13), Felt::new(14)];
+}
+
+pub fn non_fungible_asset(account_id: u64) -> Asset {
+    let non_fungible_asset_details = NonFungibleAssetDetails::new(
+        AccountId::try_from(account_id).unwrap(),
+        constants::NON_FUNGIBLE_ASSET_DATA.to_vec(),
+    )
+    .unwrap();
+    let non_fungible_asset = NonFungibleAsset::new(&non_fungible_asset_details).unwrap();
+    Asset::NonFungible(non_fungible_asset)
+}
+
+pub fn non_fungible_asset_2(account_id: u64) -> Asset {
+    let non_fungible_asset_2_details: NonFungibleAssetDetails = NonFungibleAssetDetails::new(
+        AccountId::try_from(account_id).unwrap(),
+        NON_FUNGIBLE_ASSET_DATA_2.to_vec(),
+    )
+    .unwrap();
+    let non_fungible_asset_2: NonFungibleAsset =
+        NonFungibleAsset::new(&non_fungible_asset_2_details).unwrap();
+    Asset::NonFungible(non_fungible_asset_2)
+}
```

Comment:
I wasn't planning on doing many refactors right now, otherwise the PR would be too hard to review. I think there are many functions that are in this same state so I was preparing to tackle them in the next PR.